src/parameterizations/vertical/MOM_kappa_shear.F90

```diff
@@ -794,7 +798,9 @@ subroutine kappa_shear_column(kappa, tke, dt, nzc, f2, surface_pres, hlay, dz_la
   do K=nzc,2,-1
     dist_from_bot = dist_from_bot + dz_lay(k)
     h_from_bot = h_from_bot + hlay(k)
-    I_L2_bdry(K) = ((dist_from_top(K) + dist_from_bot) * (h_from_top(K) + h_from_bot)) / &
+    !I_L2_bdry(K) = ((dist_from_top(K) + dist_from_bot) * (h_from_top(K) + h_from_bot)) / &
+    !               ((dist_from_top(K) * dist_from_bot) * (h_from_top(K) * h_from_bot))
+    I_L2_bdry(K) = 1/(CS%beta*CS%beta)*((dist_from_top(K) + dist_from_bot) * (h_from_top(K) + h_from_bot)) / &
```

Comment:
naming it 'lz_rescale' sounds good (it will remind us what it is about). Perhaps it would also be useful to add a comment right before when "I_L2_bdry(K)" is defined (on my copy, this is line 792), something like:
! Find the inverse of the squared distances from the boundaries, reduce the distance by a factor of "lz_rescale" so new_distance = old_distance * lz_rescale.